README.md

```diff
@@ -49,13 +49,9 @@ community.
 <!-- prettier-ignore-end -->
 
 1. Fork and clone the repository
-2. Ensure you have the LTS version of Node.js installed, as well as a compatible
-   version of pnpm as specified in
-   [package.json#packageManager](./package.json#L5). The easiest way to do this
-   is to install [corepack](https://github.com/nodejs/corepack) by running
-   `corepack enable`.
-3. Install the project dependencies by running `pnpm install`.
+2. Ensure you have the LTS version of Node.js installed, as well as the latest version of [Bun](https://bun.sh).
```

Comment:
maybe this should link to a specific version, kind of like how it was when the repo was still in pnpm where it linked to the `packageManager` field in the package.json